config/vufind/AccountMenu.yaml

```diff
@@ -1,89 +1,117 @@
+      checkMethod: checkHistoricloans
 
-  - name: holds
-    label: Holds and Recalls
-    route: holds-list
-    icon: user-holds
-    status: true
-    checkMethod: checkHolds
+    - name: holds
+      label: Holds and Recalls
+      route: holds-list
+      icon: user-holds
+      status: true
+      checkMethod: checkHolds
 
-  - name: storageRetrievalRequests
-    label: Storage Retrieval Requests
-    route: myresearch-storageretrievalrequests
-    icon: user-storage-retrievals
-    status: true
-    checkMethod: checkStorageRetrievalRequests
+    - name: storageRetrievalRequests
+      label: Storage Retrieval Requests
+      route: myresearch-storageretrievalrequests
+      icon: user-storage-retrievals
+      status: true
+      checkMethod: checkStorageRetrievalRequests
 
-  - name: ILLRequests
-    label: Interlibrary Loan Requests
-    route: myresearch-illrequests
-    icon: user-ill-requests
-    status: true
-    checkMethod: checkILLRequests
+    - name: ILLRequests
+      label: Interlibrary Loan Requests
+      route: myresearch-illrequests
+      icon: user-ill-requests
+      status: true
+      checkMethod: checkILLRequests
 
-  - name: fines
-    label: Fines
-    route: myresearch-fines
-    status: true
-    checkMethod: checkFines
-    iconMethod: finesIcon
+    - name: fines
+      label: Fines
+      route: myresearch-fines
+      status: true
+      checkMethod: checkFines
+      iconMethod: finesIcon
 
-  - name: profile
-    label: Profile
-    route: myresearch-profile
-    icon: profile
+    - name: profile
+      label: Profile
+      route: myresearch-profile
+      icon: profile
 
-  - name: librarycards
-    label: Library Cards
-    route: librarycards-home
-    icon: barcode
-    checkMethod: checkLibraryCards
+    - name: librarycards
+      label: Library Cards
+      route: librarycards-home
+      icon: barcode
+      checkMethod: checkLibraryCards
 
-  - name: dgcontent
-    label: Overdrive Content
-    route: overdrive-mycontent
-    icon: overdrive
-    checkMethod: checkOverdrive
+    - name: dgcontent
+      label: Overdrive Content
+      route: overdrive-mycontent
+      icon: overdrive
+      checkMethod: checkOverdrive
 
-  - name: history
-    label: Search History
-    route: search-history
-    icon: search
-    checkMethod: checkHistory
+    - name: history
+      label: Search History
+      route: search-history
+      icon: search
+      checkMethod: checkHistory
 
-  - name: logout
-    label: Log Out
-    route: myresearch-logout
-    icon: sign-out
-    checkMethod: checkLogout
+    - name: logout
+      label: Log Out
+      route: myresearch-logout
+      icon: sign-out
+      checkMethod: checkLogout
+
+Lists:
+  label: Your Lists
+  id: acc-menu-lists-header
+  MenuItems:
+    - template: myresearch/menu-mylists.phtml
+      icon: user-list
+      checkMethod: checkUserlistMode
+
+    - name: newlist
+      label: Create a List
+      route: editList
+      routeParams:
```

Comment:
This seems to be an undocumented feature; we should probably add a comment above.

Reply:
I updated the documentation. While doing so I thought that we could still add different options for URL parameters (as this one was just for dynamic routes), datafields (e.g. for lightbox) and maybe other options, but I'm not sure if that would be used or just unnecessary overhead

Reply:
Probably best to hold off on adding features until we have use cases for them... but good to keep in mind that the possibilities exist!